### PR TITLE
Added wrapper functions for data source extensions for Historic Stock Data and Weather Data

### PR DESCRIPTION
**Pull request by hidekoji**
# Description

Moved the wrapper function for Historic Stock Price and Weather Data Source wrapper to exploratory package so that these can be scheduled on exploratory.io and Collaboration Server. 

# Checklist
Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [ ] Tested with Exploratory

### CODE REVIEW

File: R/data_sourrce_extension.R [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Wrapper API for riem_measure data soruces
+get_riem_measures <- function(station = "SFO", date_start = "2020-01-01", date_end = NULL){
```
Comment (hideaki) [inline R/data_sourrce_extension.R:2]: Add `@export` like other exported functions.

File: R/data_sourrce_extension.R [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  loadNamespace("riem")
+
+  if(is.null(date_end)){
+    date_end <- as.character(Sys.Date())
+  }
+  riem::riem_measures(station = station, date_start = date_start, date_end = date_end)
+}
+
+# Wrapper API for tidyquant data source
+execute_tidyquant <- function(stocks = NULL, from = NULL, to = NULL) {
```
Comment (hideaki) [inline R/data_sourrce_extension.R:12]: Add `@export` like other exported functions.

File: R/data_sourrce_extension.R [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  }
+  if(is.null(to)){
+    to <- lubridate::today()
+  }
+
+  if (is.null(stocks) || stocks == "") {
+    # If no stocks identified, return all SP500
+    ret <- tidyquant::tq_index("SP500") %>%
+      select(1) %>%
+      tq_get(get = "stock.prices", from = from, to = to)
```
Comment (hideaki) [inline R/data_sourrce_extension.R:29]: Add tidyquant name space.

File: R/data_sourrce_extension.R [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  } else {
+    # If stocks listed, parse and return
+    ret <- stringr::str_trim(stocks) %>%
+      unlist() %>%
+      tq_get(get = "stock.prices", from = from, to = to)
```
Comment (hideaki) [inline R/data_sourrce_extension.R:34]: Add tidyquant name space.

File: R/data_sourrce_extension.R (hunk 1 above)
Comment (hidekoji) [reply R/data_sourrce_extension.R:3]: Added @export as expected.

File: R/data_sourrce_extension.R (hunk 2 above)
Comment (hidekoji) [reply R/data_sourrce_extension.R:14]: Added @export as expected.

File: R/data_sourrce_extension.R (hunk 4 above)
Comment (hidekoji) [reply R/data_sourrce_extension.R]: Added tidyquant name space.

File: R/data_sourrce_extension.R (hunk 3 above)
Comment (hidekoji) [reply R/data_sourrce_extension.R]: Added name space.

**Pull request merged**